cogs/miscellaneous/__init__.py

```diff
@@ -122,7 +122,36 @@ def __init__(self, bot):
                 color=0xF1C60C,
             )
         )
-
+        
+    @commands.command(aliases=["shorten"])
+    @locale_doc
+    async def imgur(self, ctx, given_url = None):
+        _("""Gives you a short URL from an image or long URL""")
+        if not given_url and not ctx.message.attachments:
+            return await ctx.send(_("Please supply a URL or an image attachment"))
+        if ctx.message.attachments:
+            if len(ctx.message.attachments) > 1:
+                return await ctx.send(_("Please only use one image at a time."))
+            url = ctx.message.attachments[0].url
+            if not (
+                url.endswith(".png") or url.endswith(".jpg") or url.endswith(".jpeg")
+            ):
+                return await ctx.send(_("This is not a valid image file."))
+            given_url = url
+            
+        async with self.bot.trusted_session.post(
+            "https://api.imgur.com/3/image"
+            headers={"Authorization":f"Client-ID {self.bot.config.imgur_token}"},
+            data={"image":given_url}
+        ) as r:
+            try:
+                link = (await r.json())["data"]["link"]
+            except KeyError:
+                return await ctx.send(_("Error when uploading to Imgur."))
+        await ctx.send(
+            _(f"Here's your short image URL: <{link}>")
```

Comment:
`_("Here's your short image URL: <{link}").format(link=link)`
Else it won't show properly for translation